src/Administration/Resources/app/administration/src/module/sw-order/page/sw-order-detail/index.js

```diff
@@ -137,12 +141,25 @@ export default {
             criteria
                 .addAssociation('addresses.country')
                 .addAssociation('addresses.countryState')
-                .addAssociation('deliveries.shippingMethod')
-                .addAssociation('deliveries.shippingOrderAddress')
-                .addAssociation('transactions.paymentMethod')
                 .addAssociation('documents.documentType')
                 .addAssociation('tags');
 
+            if (!Shopware.Feature.isActive('v6.8.0.0')) {
+                criteria
+                    .addAssociation('deliveries.shippingMethod')
+                    .addAssociation('deliveries.shippingOrderAddress')
+                    .addAssociation('transactions.paymentMethod');
+            } else {
+                criteria
+                    .addAssociation('primaryOrderTransaction')
+                    .addAssociation('primaryOrderTransaction.paymentMethod')
+                    .addAssociation('primaryOrderTransaction.stateMachineState')
+                    .addAssociation('primaryOrderDelivery')
+                    .addAssociation('primaryOrderDelivery.shippingMethod')
+                    .addAssociation('primaryOrderDelivery.stateMachineState')
+                    .addAssociation('primaryOrderDelivery.shippingOrderAddress.country');
+            }
```

Comment:
We should always load the primaryOrder stuff so others can migrate already
```suggestion
                .addAssociation('tags');
                .addAssociation('primaryOrderTransaction.paymentMethod')
                .addAssociation('primaryOrderTransaction.stateMachineState')
                .addAssociation('primaryOrderDelivery.shippingMethod')
                .addAssociation('primaryOrderDelivery.stateMachineState')
                .addAssociation('primaryOrderDelivery.shippingOrderAddress.country');

            if (!Shopware.Feature.isActive('v6.8.0.0')) {
                criteria
                    .addAssociation('deliveries.shippingMethod')
                    .addAssociation('deliveries.shippingOrderAddress')
                    .addAssociation('transactions.paymentMethod');
            }
```